Geometry/CMSCommonData/python/cmsExtendedGeometry2023D4XML_cfi.py

```diff
@@ -0,0 +1,129 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@boudoul can you rerun the the latest version of the geometry script to generate this file? This is a hazard of rebasing with script-generated files - the script picked up the PostLS2 beampipe changes, but the generated files did not.